### DISCUSSION TITLE
Offffff

### DISCUSSION THREAD

**Discussion by digitalborderattal** [arkypita/Vigotech-VG-L7X#5] [Ideas]
hello, i have successfully flashed my laser. Is it normal that the on / off switch after flashing only works for switching on?

thank you for your great work

**Comment by smoki3** [comment]
Nice to hear that it is working not just for me. 

Yes actually it is only working for turning on. I already looked for a solution but I haven't found it until now.

**Reply by smoki3** [reply]
@digitalborderattal 

I just pushed a commit. It is now working. 3 sec. Long press on the Power Button until you hear a sound. :)

**Reply by digitalborderattal** [reply]
läuft! Works fine!
